Plugin/src/SofaPython3/config.h

```diff
@@ -41,8 +41,7 @@
 
 // define own pybind version macro, much easier to test/read
 #define PYBIND11_SOFA_VERSION (PYBIND11_VERSION_MAJOR * 10000 \
-    + PYBIND11_VERSION_MINOR * 100 \
-    + PYBIND11_VERSION_PATCH)
+    + PYBIND11_VERSION_MINOR * 100)
 // pybind11 already bind the attributeError starting from 2.8.1 version.
 // so if the version is >= 2.8.1, macro does nothing, otherwise bind attribute_error
 #if PYBIND11_SOFA_VERSION >= 20801
```

Comment:
Actually just looking at this code makes me think that the patch version is useful.